libraries/cmake/source/lief/CMakeLists.txt

```diff
@@ -0,0 +1,388 @@
+    "${library_root}/src/PE/resources/ResourceStringTable.cpp"
+    "${library_root}/src/PE/resources/ResourceAccelerator.cpp"
+    "${library_root}/src/PE/LoadConfigurations/LoadConfigurations.tcc"
+    "${library_root}/src/PE/LoadConfigurations/LoadConfiguration.cpp"
+    "${library_root}/src/PE/LoadConfigurations/LoadConfiguration.tcc"
+    "${library_root}/src/PE/LoadConfigurations/LoadConfigurationV0.cpp"
+    "${library_root}/src/PE/LoadConfigurations/LoadConfigurationV0.tcc"
+    "${library_root}/src/PE/LoadConfigurations/LoadConfigurationV1.cpp"
+    "${library_root}/src/PE/LoadConfigurations/LoadConfigurationV1.tcc"
+    "${library_root}/src/PE/LoadConfigurations/LoadConfigurationV2.cpp"
+    "${library_root}/src/PE/LoadConfigurations/LoadConfigurationV2.tcc"
+    "${library_root}/src/PE/LoadConfigurations/LoadConfigurationV3.cpp"
+    "${library_root}/src/PE/LoadConfigurations/LoadConfigurationV3.tcc"
+    "${library_root}/src/PE/LoadConfigurations/LoadConfigurationV4.cpp"
+    "${library_root}/src/PE/LoadConfigurations/LoadConfigurationV4.tcc"
+    "${library_root}/src/PE/LoadConfigurations/LoadConfigurationV5.cpp"
+    "${library_root}/src/PE/LoadConfigurations/LoadConfigurationV5.tcc"
+    "${library_root}/src/PE/LoadConfigurations/LoadConfigurationV6.cpp"
+    "${library_root}/src/PE/LoadConfigurations/LoadConfigurationV6.tcc"
+    "${library_root}/src/PE/LoadConfigurations/LoadConfigurationV7.cpp"
+    "${library_root}/src/PE/LoadConfigurations/LoadConfigurationV7.tcc"
+    "${library_root}/src/PE/signature/Attribute.cpp"
+    "${library_root}/src/PE/signature/ContentInfo.cpp"
+    "${library_root}/src/PE/signature/Signature.cpp"
+    "${library_root}/src/PE/signature/SignerInfo.cpp"
+    "${library_root}/src/PE/signature/x509.cpp"
+    "${library_root}/src/PE/signature/OIDToString.cpp"
+    "${library_root}/src/PE/signature/SignatureParser.cpp"
+    "${library_root}/src/PE/signature/RsaInfo.cpp"
+    "${library_root}/src/PE/signature/attributes/GenericType.cpp"
+    "${library_root}/src/PE/signature/attributes/ContentType.cpp"
+    "${library_root}/src/PE/signature/attributes/SpcSpOpusInfo.cpp"
+    "${library_root}/src/PE/signature/attributes/MsCounterSign.cpp"
+    "${library_root}/src/PE/signature/attributes/MsSpcNestedSignature.cpp"
+    "${library_root}/src/PE/signature/attributes/MsSpcStatementType.cpp"
+    "${library_root}/src/PE/signature/attributes/PKCS9AtSequenceNumber.cpp"
+    "${library_root}/src/PE/signature/attributes/PKCS9CounterSignature.cpp"
+    "${library_root}/src/PE/signature/attributes/PKCS9MessageDigest.cpp"
+    "${library_root}/src/PE/signature/attributes/PKCS9SigningTime.cpp"
+    )
+    target_include_directories(thirdparty_lief PRIVATE
+    "${library_root}/src/PE/utils/ordinals_lookup_tables"
+    "${library_root}/src/PE/utils/ordinals_lookup_tables_std"
+    "${library_root}/src/PE/signatures/"
+    )
+    if(MSVC)
+      add_definitions(-DNOMINMAX)
```

Comment:
Do not use the global `add_definitions`, use the target specific `target_compiled_definitions` instead.

Reply:
good to know, removed.